src/wireviz/Harness.py

```diff
@@ -253,6 +253,9 @@ def create_graph(self) -> Graph:
 
                 pinhtml.append("  </table>")
 
+                if len(pinhtml) == 2:  # Table start and end with no rows between?
+                    pinhtml = ["<!-- all pins hidden -->"]  # Avoid Graphviz error
+
```

Comment:
Not really. Most users will not look at the HTML source, and hence, will not see this comment.

However, it might be helpful when comparing HTML files to see the effect of certain YAML input changes.

Reply:
No error message is needed IMHO.